openpype/modules/sync_server/launch_hooks/pre_copy_last_published_workfile.py

```diff
@@ -194,3 +195,71 @@ def execute(self):
         self.data["last_workfile_path"] = local_workfile_path
         # Keep source filepath for further path conformation
         self.data["source_filepath"] = last_published_workfile_path
+
+        # Get resources directory
+        resources_dir = os.path.join(
+            os.path.dirname(local_workfile_path), 'resources'
+        )
+        # Make resource directory if it doesn't exist
+        if not os.path.exists(resources_dir):
+            os.mkdir(resources_dir)
+
+        # Copy resources to the local resources directory
+        for file in workfile_representation['files']:
+            # Get resource main path
+            resource_main_path = file["path"].replace(
+                "{root[main]}", str(anatomy.roots["main"])
+            )
```

Comment:
@kalisp 

Why is this a hardcoded `root[main]`? Couldn't the studio template technically have _any_ root set? This should have been `file["path"].format(**anatomy.roots)` right?

Reply:
I thought I fixed that but I guess I must have made a mistake at some point because the commit is gone for some reason. Dunno what happened but I have other another fix to do anyway (Found cases in which workfiles are copied in the `resources` folder while they shouldn't) so I'll bring those changes back.

Reply:
Fix available in PR #5733